Solutions/Ais.Net.Receiver.Host.Console/Ais/Net/Receiver/Host/Console/Program.cs

```diff
@@ -28,14 +34,29 @@ public static class Program
     /// <summary>
     /// Entry point for the application.
     /// </summary>
+    /// <param name="args">Command line arguments.</param>
     /// <returns>Task representing the operation.</returns>
-    public static async Task Main()
+    public static async Task Main(string[] args)
```

Comment:
Since you're changing this, it also feels like another candidate for the low-ceremony entry point syntax, since as far as I can tell this is a one-method type. (That automatically makes `args` available to you without needing a declaration, as well as removing all the other boilerplate.)